src/model/SubSurface.hpp

```diff
@@ -263,6 +263,12 @@ namespace model {
 
     boost::optional<AirflowNetworkSurface> airflowNetworkSurface() const;
 
+    /* Get the total area of the sub surface including frame and divider area */
+    double roughOpeningArea() const;
+
+    /* Get the rough opening vertices for the subsurface */
+    std::vector<Point3d> roughOpening() const;
```

Comment:
```suggestion
    std::vector<Point3d> roughOpeningVertices() const;
```

Reply:
This could be implemented to point at a `grossAreaVertices` method on `PlanarSurface`, `grossAreaVertices` could be used to implement `PlanarSurface::overlaps(const PlanarSurface& other) const`